src/main/java/it/gov/pagopa/payhub/activities/activity/paymentsreporting/PaymentsReportingIngestionFlowFileActivityImpl.java

```diff
@@ -19,15 +22,16 @@
 public class PaymentsReportingIngestionFlowFileActivityImpl implements PaymentsReportingIngestionFlowFileActivity {
 	private final IngestionFlowFileDao ingestionFlowFileDao;
 	private final IngestionFlowFileRetrieverService ingestionFlowFileRetrieverService;
-
+	private final FlussoRiversamentoHandler flussoRiversamentoHandler;
 
 	public PaymentsReportingIngestionFlowFileActivityImpl(IngestionFlowFileDao ingestionFlowFileDao,
-	                                                      IngestionFlowFileRetrieverService ingestionFlowFileRetrieverService) {
+	                                                      IngestionFlowFileRetrieverService ingestionFlowFileRetrieverService,
+	                                                      FlussoRiversamentoHandler flussoRiversamentoHandler) {
 		this.ingestionFlowFileDao = ingestionFlowFileDao;
 		this.ingestionFlowFileRetrieverService = ingestionFlowFileRetrieverService;
+		this.flussoRiversamentoHandler = flussoRiversamentoHandler;
 	}
 
-
 	@Override
 	public PaymentsReportingIngestionFlowFileActivityResult processFile(Long ingestionFlowFileId) {
 		List<String> iufList = new ArrayList<>();
```

Comment:
the Activity interface has been designed to be flexible in order to return a list of IUF, as I see, the only importable format here will always return just a single IUF.
Don't create an ArrayList, just use List.of with the only available iuf (or Collections.emptyList() if there are no iuf)